core/src/test/scala/unit/kafka/integration/SecurityTestHarness.scala

```diff
@@ -0,0 +1,153 @@
+/**
```

Comment:
This class is not used in this version of the PR. If the other changes are fine, then we can get rid of this class.